internal/client/project.go

```diff
@@ -53,6 +62,9 @@ func New(ctx context.Context, opts ...Option) (*Project, error) {
 		}
 	}
 
+	// Used by any background goroutines that we'll spawn (like runner job processing)
+	client.bg, client.cancel = context.WithCancel(context.Background())
```

Comment:
can we pair the names so its clear what is being canceled here? i.e. `bgCancel` instead of `cancel`? Right now we have one context so its fine, but I sometimes get confused as projects mature.